enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -79,6 +80,8 @@ Please refer to `/var/cfengine/output`, `/var/log/postgresql.log` and  `/opt/cfe
 when problems occur. Sending these logs to us in bug reports will help significantly as we fine tune the Federated
 Reporting feature.
 
+Also see [Disable Feeder] for information about how to temporarily disable a feeder's participation in Federated Reporting in case that is causing an issue for the Feeder Hub.
```

Comment:
@nickanderson is this how you reference inside a doc?

Reply:
I think it's `[Disable Feeder](#Disable Feeder)`